deployment/refinery-modules/refinery/manifests/docker_.pp

```diff
@@ -0,0 +1,5 @@
+class refinery::docker_ {
+  class { 'docker':
```

Comment:
This should be `class { '::docker':` [to indicate global namespace explicitly](http://serverfault.com/questions/421023/why-does-my-namespace-and-class-name-conflict)